README.md

```diff
@@ -296,6 +475,24 @@ small extension to that program in a way that virtually guarantees some sort
 of merge conflict, so everyone will have the experience of dealing with
 conflicts.
 
+### Open GitKraken
```

Comment:
Might want to include some instruction on what to do when initially opening GitKraken. Students will want to sign in with GitHub and they should have already signed up for the GitHub Student Pack to get the pro version of GitKraken (for it to actually have merge conflict support). Additionally it will ask to setup their git info (like they already did in the command line) and they should enter their name and whatever email they used for GitHub.

Reply:
Thanks. This is the problem of already using the tool – I couldn't remember what it looked like to use it for the first time. :)

Reply:
I think that e67a9c6 addresses this.